specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -441,41 +441,33 @@
           }
         ],
         "responses": {
-          "201": {
-            "description": "Success",
-            "schema": {
-              "type": "string"
-            }
-          },
-          "default": {
-            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
-            "schema": {
-              "$ref": "#/definitions/ErrorResponse"
-            },
-            "examples": {
-              "application/json": {
-                "error": {
-                  "code": "BadArgument",
-                  "innerError": {
-                    "requestId": "00000000-0000-0000-0000-000000000000"
-                  },
-                  "message": "Request body is invalid."
+            "201": {
+              "description": "Request is queued successfully.",
+              "headers": {
+                "Location": {
+                  "type": "string",
+                  "description": "Location and ID of the composed model. The status of composed model is specified in the status property at the model location."
                 }
               }
+            },
+            "default": {
+              "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+              "schema": {
+                "$ref": "#/definitions/ErrorResponse"
+              }
             }
-          }
         },
         "x-ms-examples": {
-          "Compose custom models": {
-            "$ref": "./examples/ComposeModels.json"
-          }
+            "Compose custom models": {
+              "$ref": "./examples/ComposeModels.json"
+            }
         }
       }
-    },    
+    },
     "/prebuilt/businessCard/analyze": {
       "post": {
         "summary": "Analyze Business Card",
-        "description": "Extract field text and semantic values from a given business card document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png' or 'image/tiff'. Alternatively, use 'application/json' type to specify the location (Uri or local path) of the document to be analyzed.",
+        "description": "Extract field text and semantic values from a given business card document. The input document must be of one of the supported content types - 'application/pdf', 'image/jpeg', 'image/png' or 'image/tiff'. Alternatively, use 'application/json' type to specify the location (Uri) of the document to be analyzed.",
```

Comment:
The "local path" reference is actually sprinkle across all analyze operations, not just for receipts.